torch/csrc/jit/ir/named_value.h

```diff
@@ -70,6 +68,12 @@ struct NamedValue {
     return *loc_;
   }
 
+  SourceRange locOr(const SourceRange& backup_location) const {
+    if (!loc_)
+      return backup_location;
+    return loc();
+  }
+
```

Comment:
Looks like this was moved from L48? Any reason for this?

Reply:
It is nicer to have setter and getter together. But please save pure clean-ups like this for separate PR.

Think about current situation: review needed to analyze what this is for and then compare whether this is just moved or changed and moved.

The alternative situation: leave this method untouched in current PR so there is no additional workload for this review. Have another separate PR that's titled as "move locOr closer to corresponding setter", and the PR is only about moving code. This separate follow-up PR requires no effort to review. This alternative situation (approach) makes both PRs easier to review while making the code organized better.

Reply:
Thanks everyone for explaining best practices to me! It sounds corny to say this, but \~\*\*feedback is a gift**~